src/tooling/piral-cli/src/apps/build-piral.ts

```diff
@@ -160,6 +173,14 @@ export async function buildPiral(baseDir = process.cwd(), options: BuildPiralOpt
       ignored,
     });
 
+    // run post piral build script if provided
+    if (postPiralBuildRelease) {
+      progress(`Running 'postPiralBuildRelease' script ...`);
+      logInfo(`Run: ${postPiralBuildRelease}`);
+      await runScript(postPiralBuildRelease, root);
+      logInfo(`Successfully ran 'postPiralBuildRelease' script.`);
```

Comment:
This is also just a debug output.

(the same two comments about logging apply to the other block).